cpu/cc3200/Makefile.features

```diff
@@ -0,0 +1,8 @@
+include $(RIOTCPU)/cortexm_common/Makefile.features
+
+
+# Additional MCU features
+FEATURES_PROVIDED += periph_cpuid
+FEATURES_PROVIDED += periph_hwrng
+FEATURES_PROVIDED += puf_sram
```

Comment:
Is this implemented?

Reply:
- cpuid is ported don't know why I removed it from the pull request (will add it back in)
- hwrng is gone for the cc3200 so I will remove it (it can be accessed over the Network co-processor but it is a quite hacky). I will remove that feature. Thank you for noticing :)
- I thought sram puf does not need a cpu specific implementation but is more of a ram property

Reply:
Oh yea the sram puf really just reads uninitialized SRAM - I thought it was a more elaborate *physical unclonable function* that some chips might provide - but this basically just say 'the ram on this platform is pretty much random at startup, so it can be used as a random seed'.

I would have expected this to be mostly 0, but apparently [it's a think](https://en.wikipedia.org/wiki/Types_of_physical_unclonable_function#SRAM_PUF) - so keep it.

If it's really just that it might be moved to `cortexm_common/` at some point.